pkg/tenant/members_test.go

```diff
@@ -360,8 +360,6 @@ func (suite *tenantTestSuite) TestMemberDetail() {
 		switch in.Namespace {
 		case db.MembersNamespace:
 			return &pb.GetReply{Value: data}, nil
-		case db.OrganizationNamespace:
```

Comment:
I removed mentions of `OrganizationNamespace` since org verification has been removes from the member handlers.